src/api/users.ts

```diff
@@ -13,26 +14,10 @@ const {ACCESS_TOKEN_SECRET} = process.env;
 
 const usersRouter = express.Router();
 
-// GET /api/users
-//TODO: Delete or restrict this endpoint to admin or logged in user.
-//TODO: Strip password for users object if this route is kept.
-
-usersRouter.get("/", async (req, res, next): Promise<void> => {
-    try {
-        const users = await prisma.user.findMany();
-        res.send({users});
-    } catch (e) {
-        next(e);
-    }
-})
 
 // GET /api/users/me
-//TODO: require user 
-usersRouter.get("/me", async (req, res, next): Promise<void> => {
-    if (!req.user) {
-        res.status(401);
-        next({name: "NotLoggedIn", message: "Must be logged in to access this route."});
-    } else {
+usersRouter.get("/me", requireUser, async (req, res, next): Promise<void> => {
+    if (req.user) {
```

Comment:
Do you need to do this? It is also checked in `requireUser` - but maybe it's needed to make TS happy?

Reply:
It was indeed to appease TS.  Because it was not understanding that the `requireUser` middleware would only allow the route's code to execute if req.user existed.